.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: python
 install:
   - pip install coveralls
+  - pip install mock
```

Comment:
Have you checked if this branch is installable via `pip`? I think that we might need `requirements.txt` to make it installable, but I'm not very sure, so please check it.